axelrod/strategy_transformers.py

```diff
@@ -84,23 +92,18 @@ def __call__(self, PlayerClass):
             except KeyError:
                 pass
 
-            # Is the original strategy method a static method?
-            signature = inspect.signature(PlayerClass.strategy)
-            strategy_args = [p.name for p in signature.parameters.values()
-                    if p.kind == inspect.Parameter.POSITIONAL_OR_KEYWORD]
-            is_static = True
-            if len(strategy_args) > 1:
-                is_static = False
-
             # Define the new strategy method, wrapping the existing method
             # with `strategy_wrapper`
             def strategy(self, opponent):
-
-                if is_static:
-                    # static method
-                    proposed_action = PlayerClass.strategy(opponent)
+                if strategy_wrapper == dual_wrapper:
```

Comment:
it is.  but more importantly, it's avoiding calling the wrapped player twice.  since dual wrapper requires a `proposed_action` variable, but doesn't use it, the `C` is just a stand-in value.